Gemfile.lock

```diff
@@ -375,23 +381,24 @@ GEM
     thread_safe (0.3.5)
     tilt (2.0.5)
     ttfunk (1.4.0)
-    turbolinks (2.5.3)
-      coffee-rails
+    turbolinks (5.0.1)
```

Comment:
The date picker doesn't work correctly. It needs a full refresh to load... 😒
`wice_grid` works as before.